docs/reference/remap.cue

```diff
@@ -179,19 +185,31 @@ remap: #Remap & {
 			title: "Invalid argument type"
 			input: log: not_a_string: 1
 			source: """
-				upcase(.not_a_string)
+				upcase(42)
 				"""
 			raises: compiletime: """
-				error: invalid argument type
-				  ┌─ :1:1
+				error[E110]: invalid argument type
+				  ┌─ :1:8
 				  │
-				1 │ upcase(.not_a_string)
-				  │        ^^^^^^^^^^^^^
+				1 │ upcase(42)
+				  │        ^^
 				  │        │
-				  │        this expression resolves to unknown type
+				  │        this expression resolves to the exact type "integer"
 				  │        but the parameter "value" expects the exact type "string"
 				  │
-				  = see language documentation at: https://vector.dev/docs/reference/vrl/
+				  = try: ensuring an appropriate type at runtime
+				  =
+				  =     42 = string!(42)
+				  =     upcase(42)
+				  =
+				  = try: coercing to an appropriate type and specifying a default value as a fallback in case coercion fails
+				  =
+				  =     42 = to_string(42) ?? "default"
+				  =     upcase(42)
```

Comment:
I'll create a follow-up PR to remove this suggestion, since they no longer make sense, because an "invalid argument type" error now only happens when there's no way a type can be coerced into the required parameter type.